src/temporal/client/schedule.clj

```diff
@@ -0,0 +1,176 @@
+(ns temporal.client.schedule
+  (:require [taoensso.timbre :as log]
+            [temporal.client.options :as copts]
+            [temporal.internal.grpc :as g]
+            [temporal.internal.utils :as u]
+            [temporal.internal.schedule :as s])
+  (:import [java.time Duration]
+           [io.temporal.client.schedules ScheduleClient ScheduleUpdate ScheduleUpdateInput]))
+
+(set! *warn-on-reflection* true)
+
+(defn create-client
+  "Creates a `ScheduleClient` instance suitable for interacting with Temporal's Schedules.
+   The `ScheduleClient` has slightly less options then the `WorkflowClient` but it is mostly the same otherwise.
+
+   Arguments:
+
+   - `options`: Client configuration option map (See below)
+   - `timeout`: Connection timeout as a [Duration](https://docs.oracle.com/javase/8/docs/api//java/time/Duration.html) (default: 5s)
+
+    #### options map
+
+
+   | Value                     | Description                                                                 | Type         | Default |
```

Comment:
Overall, code looks great, ty.  My only thought is whether there is an opportunity to DRY up the option definition.

My first question on that front would be: Are the options actually identical to wf-options? If so, we can define them in one place and simply refer to the other place here with a [[ ]] style link.  Ill try to dig up an example of what I mean.

Reply:
This is an example of what I meant: 

https://cljdoc.org/d/io.github.manetu/temporal-sdk/0.19.0/api/temporal.client.worker#start

Reply:
I will take a stab at this tomorrow but the `ScheduleClient` uses a different static option type then the other `WorkflowClient`

The [ScheduleClient](https://www.javadoc.io/doc/io.temporal/temporal-sdk/latest/io/temporal/client/schedules/ScheduleClient.html) uses the [ScheduleClientOptions](https://www.javadoc.io/doc/io.temporal/temporal-sdk/latest/io/temporal/client/schedules/ScheduleClientOptions.html) while the [WorkflowClient](https://www.javadoc.io/doc/io.temporal/temporal-sdk/latest/io/temporal/client/WorkflowClient.html) uses the [WorkflowClientOptions](https://www.javadoc.io/doc/io.temporal/temporal-sdk/latest/io/temporal/client/WorkflowClientOptions.html) even though they both take the same fields it seems which is the builder for those few options are different.

I made the `internal/grpc.clj` file because the options were definitely shared because the constructors for both the `WorkflowClient` and `ScheduleClient` take a `io.temporal.serviceclient.WorkflowServiceStubs` as their first parameter. 

I think I could just move the majority of those options documentation the to `internal/grpc.clj` and that would get rid of the duplication for the clients. I will look deeper into what you recommend for the worker as that might help.